generate/templates/templates/nodegit.js

```diff
@@ -52,19 +54,55 @@ exports.__proto__ = rawApi;
 
 var importExtension = function(name) {
   try {
-    require("./" + name);
-  } catch (unhandledException) {}
+    require(local(name));
```

Comment:
Gotcha. I've just been adding it everywhere in the name of consistency. I was actually having issues with files not loading thinking it wasnt grabbing htem, but they were bing swallowed by the catch on line 59